oggm/params.cfg

```diff
@@ -58,6 +58,9 @@ mpi_recv_buf_size = 131072
 # Check for the integrity of the files OGGM downloads at run time
 dl_verify = True
 
+# Default number of files to be cached in the temporary directory
+LRU_maxsize = 100
```

Comment:
Can you change this to all low-caps? Also in cfg.PARAMS - I think we only have low-caps in there